dawn/src/dawn/SIR/SIR.h

```diff
@@ -297,12 +297,20 @@ struct VerticalRegion {
   SourceLocation Loc;                         ///< Source location of the vertical region
   std::shared_ptr<sir::AST> Ast;              ///< AST of the region
   std::shared_ptr<Interval> VerticalInterval; ///< Interval description of the region
-  LoopOrderKind LoopOrder;                    /// Loop order (usually associated with the k-loop)
+  LoopOrderKind LoopOrder;                    ///< Loop order (usually associated with the k-loop)
+  /// If it is not instantiated, iteration over the full domain is assumed.
+  std::array<std::optional<Interval>, 2> iterationSpace_; /// < Iteration space in the horizontal.
 
   VerticalRegion(const std::shared_ptr<sir::AST>& ast,
                  const std::shared_ptr<Interval>& verticalInterval, LoopOrderKind loopOrder,
                  SourceLocation loc = SourceLocation())
       : Loc(loc), Ast(ast), VerticalInterval(verticalInterval), LoopOrder(loopOrder) {}
+  VerticalRegion(const std::shared_ptr<sir::AST>& ast,
```

Comment:
Adding this constructor does not conform to the new coding standards set by @mroethlin.

Reply:
:+1: